macros/edr/data_monitoring/monitors_query/table_monitoring_query.sql

```diff
@@ -1,6 +1,6 @@
 {% macro table_monitoring_query(monitored_table_relation, timestamp_column, is_timestamp, min_bucket_start, table_monitors, freshness_column=none) %}
 
-    {%- set max_bucket_end = "'"~ run_started_at.strftime("%Y-%m-%d 00:00:00")~"'" %}
+    {%- set max_bucket_end = "'"~ elementary.get_run_started_at().strftime("%Y-%m-%d 00:00:00")~"'" %}
```

Comment:
Should we do this change for the column_monitoring_query as well?
If not it means we can't tests it on the e2e

Reply:
What do you think about doing it for all the run_started_at usages?